data/rule_data.yml

```diff
@@ -20,3 +20,7 @@ rule_data:
   allowed_java_component_sources:
   - redhat
   - rebuilt
+
+  # Defaults for reference. Levels are "critical", "high", "medium", and "low".
```

Comment:
What about the other defaults in `policy/lib/rule_data.rego`? Do we need to mention them here as well? It's a bit confusing how there are two places to provide "default" values. So maybe, at least, we can mention here that some other defaults are in `policy/lib/rule_data.rego`?

Reply:
Yeah maybe. It's a bit inconsistent now. Will move to draft to consider.

Reply:
Maybe documenting the defaults nicely is a better way to do it.

Reply:
FWIW, I thought I did document this sufficiently on the rule description, but maybe that's a good place to start?